app/src/main/java/pl/pola_app/ui/fragment/ProductDetailsFragment.java

```diff
@@ -132,36 +131,45 @@ public void onActivityCreated(Bundle savedInstanceState) {
             plScoreText.setText("?");
         }
 
-        if(product.company.plCapital != null) {
-            plCapitalBar.setProgress(product.company.plCapital);
-            plCapitalText.setText(product.company.plCapital + "%");
+        if(product.plCapital != null) {
+            plCapitalBar.setProgress(product.plCapital);
+            plCapitalText.setText(product.plCapital + "%");
         } else {
             plCapitalBar.setProgress(0);
             plCapitalText.setText("?");
         }
 
-        if (product.company.plWorkers != null && product.company.plWorkers != 0) {
-            buttonWorkers.setSelected(true);
-        } else if (product.company.plWorkers == null) {
-            buttonWorkers.setEnabled(false);
-        }
+        if(product.altText != null) {
+            plDataLayout.setVisibility(View.GONE);
+            altText.setVisibility(View.VISIBLE);
+            altText.setText(product.altText);
```

Comment:
when altText is not null it should be only visible view (check box's, capital should bot not visible)